.github/CHANGELOG.md

```diff
@@ -188,7 +188,7 @@ v0.15.1 is an unplanned release to fix a XSS security vulnerability.
 - Added portable zip archive for both x86 and x64 Windows.
 - Changed `viewToggleFullScreen` and `windowCloseWindow` key bindings to `windowToggleFullScreen` and `fileCloseWindow`.
 - Removed `viewChangeFont` key binding.
-- Mark Text is now single-instance application on Linux and Windows too.
```

Comment:
I think changelog should stay unchanged.

Reply:
Fair enough I'll revert that